Increase the tolerance for a test that is known to be a little flaky
https://github.com/ginkgo-project/ginkgo/issues/1866

diff -ruN ginkgo-1.9.0.orig/reference/test/solver/cgs_kernels.cpp ginkgo-1.9.0/reference/test/solver/cgs_kernels.cpp
--- ginkgo-1.9.0.orig/reference/test/solver/cgs_kernels.cpp	2024-12-09 14:42:38.000000000 +0000
+++ ginkgo-1.9.0/reference/test/solver/cgs_kernels.cpp	2025-06-12 12:06:42.174648062 +0000
@@ -345,7 +345,7 @@
     GKO_ASSERT_MTX_NEAR(x,
                         l({value_type{-4.0, 8.0}, value_type{-1.0, 2.0},
                            value_type{4.0, -8.0}}),
-                        (r_mixed<value_type, TypeParam>() * 1e2));
+                        (r_mixed<value_type, TypeParam>() * 3e2));
 }
 
 
